### PR TITLE
New york time index page

### PR DESCRIPTION
**Pull request by rukundoeric**
### What does this do? 
*  Floating and positioning elements
### Description of Task to be completed? 
* A structural clone of the New York Times page.
### How should this be manually tested? 
* Clone this repo https://github.com/rukundoeric/Positioning-and-Floating-Elements/tree/nyt-index-page
* Navigate to project directory.
* If you have a live server in you vs-code, just run it.
* Otherwise just open the index page with your browser.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,486 @@
...
+            text-decoration: none; }
+            .main-container .article-content.with-aside aside .aside-list .item a:hover {
+              text-decoration: underline; }
+  .main-container .p-links {
+    color: #326891 !important; }
+    .main-container .p-links:hover {
+      text-decoration: none; }
+  .main-container .article-content > div {
+    width: 45%;
+    /* stylelint-disable */
```
Comment (umairarshadbutt) [inline css/style.css]: Remove comments

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,486 @@
...
+      .main-container .article-content > div figure figcaption {
+        color: #333;
+        font-size: 17px; }
+        .main-container .article-content > div figure figcaption span {
+          color: #6b6666 !important;
+          font-size: 15px; }
+  .main-container .article-image div {
+    width: 70%;
+    padding: 0;
+    /* stylelint-disable */
```
Comment (umairarshadbutt) [inline css/style.css]: Remove comments

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,486 @@
...
+        color: #6b6666;
+        line-height: 25px; }
+        .main-container .article-image div figure figcaption span {
+          color: #a3a3a3;
+          font-size: 14px; }
+  .main-container .article-image.telescop div {
+    width: 50% !important; }
+  .main-container .conclusion-content {
+    padding-bottom: 100px;
+    /* stylelint-disable */
```
Comment (umairarshadbutt) [inline css/style.css]: Remove comments

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,486 @@
...
+            margin-left: 20px; }
+  .main-container .more-readings {
+    width: 75%;
+    display: grid;
+    grid-template-columns: 70% 30%; }
+    .main-container .more-readings .headers {
+      margin: 0 15px;
+      padding: 25px 0 15px 0;
+      border-top: 3px solid black;
+      /* stylelint-disable */
```
Comment (umairarshadbutt) [inline css/style.css]: Remove comments

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,486 @@
...
+        font-size: 19px;
+        font-weight: 700; }
+    .main-container .more-readings .readings-left {
+      display: grid;
+      grid-template-columns: 33.3% 33.3% 33.3%; }
+      .main-container .more-readings .readings-left .item-container {
+        width: 100%;
+        padding: 15px;
+        box-sizing: border-box;
+        /* stylelint-disable */
```
Comment (umairarshadbutt) [inline css/style.css]: Remove comments

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,486 @@
...
+      padding: 0 30px 25px 20px; }
+      .main-container .more-readings .most-popular ul {
+        list-style: none;
+        padding: 0 13px; }
+        .main-container .more-readings .most-popular ul li {
+          margin: 0;
+          list-style: none;
+          padding: 20px 0;
+          border-top: 3px solid #d1d1d1;
+          /* stylelint-disable */
```
Comment (umairarshadbutt) [inline css/style.css]: Remove comments

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,486 @@
...
+            font-size: 15px; }
+  .main-container section {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    /* stylelint-disable */ }
+    .main-container section:nth-of-type(2) {
+      /* stylelint-disable */
+      /* stylelint-enable */ }
```
Comment (umairarshadbutt) [inline css/style.css]: Remove comments

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,486 @@
...
+      .main-container section:nth-of-type(2) div {
+        padding: 20px 0 0 0;
+        width: 45%; }
+        .main-container section:nth-of-type(2) div h1 {
+          font-family: "Poppins", sans-serif;
+          font-size: 40px;
+          font-style: italic;
+          line-height: 60px; }
+    .main-container section:nth-of-type(4) {
+      /* stylelint-enable */ }
```
Comment (umairarshadbutt) [inline css/style.css]: Remove comments

File: css/style.css [hunk 9]
```diff
@@ -0,0 +1,486 @@
...
+              width: 40px;
+              margin-left: 20px; }
+  .main-container .expand {
+    background-color: #fff;
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    border-top: 1px solid black;
+    padding: 30px 0;
+    /* stylelint-disable */
```
Comment (umairarshadbutt) [inline css/style.css]: Remove comments

File: css/style.css [hunk 10]
```diff
@@ -0,0 +1,486 @@
...
+        .main-container .expand div:nth-of-type(2) i {
+          font-size: 20px;
+          font-weight: 800; }
+  .main-container .theory {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    /* stylelint-disable */
+    /* stylelint-enable */ }
```
Comment (umairarshadbutt) [inline css/style.css]: Remove comments

File: css/style.css [hunk 11]
```diff
@@ -0,0 +1,486 @@
...
+footer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column; }
+  footer .footer-content {
+    width: 85%;
+    padding: 20px 0;
+    /* stylelint-disable */
+    /* stylelint-enable */ }
```
Comment (umairarshadbutt) [inline css/style.css]: Remove comments

File: css/style.css [hunk 12]
```diff
@@ -0,0 +1,486 @@
...
+        font-size: 12px;
+        text-decoration: none;
+        color: #326891; }
+    footer .footer-content .links {
+      display: grid;
+      grid-template-columns: repeat(6, 1fr);
+      padding-bottom: 5px;
+      border-bottom: 1px solid #c5c5c5;
+      /* stylelint-disable */
+      /* stylelint-enable */ }
```
Comment (umairarshadbutt) [inline css/style.css]: Remove comments

File: css/style.css [hunk 13]
```diff
@@ -0,0 +1,486 @@
...
+          padding: 5px 0;
+          border-bottom: 1px solid #c5c5c5 !important; }
+        footer .footer-content .links .sub-item .sub_small {
+          font-size: 12px; }
+    footer .footer-content .sub-section {
+      text-align: center;
+      font-size: 13px;
+      margin: 20px 0 60px 0;
+      /* stylelint-disable */
+      /* stylelint-enable */ }
```
Comment (umairarshadbutt) [inline css/style.css]: Remove comments

File: css/style.css [hunk 14]
```diff
@@ -0,0 +1,486 @@
...
+  padding: 0;
+  margin: 0; }
+
+.nav-bar {
+  position: fixed;
+  width: 100%;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 10px !important;
```
Comment (umairarshadbutt) [inline css/style.css]: Remove `!important`

File: css/style.css [hunk 15]
```diff
@@ -0,0 +1,486 @@
...
+  border: 1px solid transparent; }
+  .main-container .advertisement {
+    text-align: center;
+    background-color: #f8f8f8;
+    border: 1px solid #f8f8f8;
+    padding-bottom: 30px; }
+    .main-container .advertisement h6 {
+      font-size: 10px;
+      color: #c5c5c5;
+      font-weight: 200 !important;
```
Comment (umairarshadbutt) [inline css/style.css]: Remove `!important`

File: css/style.css [hunk 16]
```diff
@@ -0,0 +1,486 @@
...
+      font-family: helvetica, arial, sans-serif; }
+    .main-container .advertisement img {
+      max-width: 80%;
+      max-height: 500px; }
+  .main-container .article-content.with-aside {
+    position: relative !important; }
```
Comment (umairarshadbutt) [inline css/style.css]: Remove `!important`

File: css/style.css [hunk 17]
```diff
@@ -0,0 +1,486 @@
...
+            width: 70px; }
+          .main-container .article-content.with-aside aside .aside-list .item a {
+            margin-left: 10px;
+            font-size: 14px;
+            color: black;
+            text-decoration: none; }
+            .main-container .article-content.with-aside aside .aside-list .item a:hover {
+              text-decoration: underline; }
+  .main-container .p-links {
+    color: #326891 !important; }
```
Comment (umairarshadbutt) [inline css/style.css]: Remove `!important`

File: css/style.css [hunk 18]
```diff
@@ -0,0 +1,486 @@
...
+        border-bottom: 2px solid #e2e2e2;
+        margin: 20px 0; }
+      .main-container .article-content > div .subscribe label {
+        color: black;
+        font-weight: 600; }
+      .main-container .article-content > div .subscribe a {
+        text-decoration: none;
+        color: #326891; }
+    .main-container .article-content > div figure {
+      margin: 0 !important; }
```
Comment (umairarshadbutt) [inline css/style.css]: Remove `!important`

File: css/style.css [hunk 19]
```diff
@@ -0,0 +1,486 @@
...
+      .main-container .article-content > div figure iframe {
+        width: 100%;
+        border: 0;
+        height: 400px; }
+      .main-container .article-content > div figure figcaption {
+        color: #333;
+        font-size: 17px; }
+        .main-container .article-content > div figure figcaption span {
+          color: #6b6666 !important;
```
Comment (umairarshadbutt) [inline css/style.css]: Remove `!important`

File: css/style.css [hunk 20]
```diff
@@ -0,0 +1,486 @@
...
+      .main-container .article-image div figure figcaption {
+        font-size: 15px;
+        font-family: georgia, "times new roman", times, serif;
+        color: #6b6666;
+        line-height: 25px; }
+        .main-container .article-image div figure figcaption span {
+          color: #a3a3a3;
+          font-size: 14px; }
+  .main-container .article-image.telescop div {
+    width: 50% !important; }
```
Comment (umairarshadbutt) [inline css/style.css]: Remove `!important`

File: css/style.css [hunk 21]
```diff
@@ -0,0 +1,486 @@
...
+            height: 18px;
+            width: 18px; }
+          footer .footer-content .links .item ul a {
+            color: black;
+            text-decoration: none; }
+            footer .footer-content .links .item ul a:hover {
+              text-decoration: underline; }
+      footer .footer-content .links .sub-item {
+        padding: 0 20px;
+        border-left: 1px solid #c5c5c5 !important; }
```
Comment (umairarshadbutt) [inline css/style.css]: Remove `!important`

File: css/style.css [hunk 22]
```diff
@@ -0,0 +1,486 @@
...
+          color: black;
+          font-weight: 900;
+          font-size: 16px;
+          margin: 4px 0; }
+          footer .footer-content .links .sub-item .sub:nth-of-type(4) {
+            margin-bottom: 15px; }
+        footer .footer-content .links .sub-item .underline {
+          margin-bottom: 10px;
+          padding: 5px 0;
+          border-bottom: 1px solid #c5c5c5 !important; }
```
Comment (umairarshadbutt) [inline css/style.css]: Remove `!important`

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @rukundoeric AND @Lameck1 
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly work on the suggested changes  👇.
- [x] Use a single image in  **The Theory of Inflation** section([Image](https://static01.nyt.com/newsgraphics/2014/03/18/theory-of-inflation/1ebaa1b972edda0f407fbaa0e010f6b50fb6792e/inflation-945.png))
- [x] Use image  URL in image src attribute in **Editors' Picks**
Example: 
``
- [x] Remove comments and `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)

- [x] Fix HTML validation [HTML Validation Screenshort](https://user-images.githubusercontent.com/29689273/94264086-82317500-ff4f-11ea-88fd-a60f78d70421.png). You can check HTML validations in [validator.w3.org](https://validator.w3.org/)




##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by javitocor — CHANGES_REQUESTED** [review]
#STATUS CHANGES REQUIRED ♻️ 

Great job with the changes 👍 
However, still some more changes to do before approving the project
Check my comments below 👇  and resubmit the project for a new code review ⏩ 

Good luck, team 😃 

- The NY times logo on the header must be a link, please fix it    
   - The NY Times logo is a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))

- Kindly modify the theory of inflation section, now the text overlap the images
Capture

- Kindly, modify the most popular section, must be similar to the original, now it is only half of its width
Capture2
Capture1

[OPTIONAL]
- Add cursor pointer to the social media icons and header buttons
- You can use `section` tag and not repeat `div` that often, it is a good practice to use semantic HTML tags

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

#### Hello, @rukundoeric  and @Lameck1 👋   

You've done an awesome job so far 👌. I see you have fixed the *The theory of inflation* section, well done. Just make these few changes and you will be good to go.

### Please only submit for Code Review Request when you are ready, I noticed you were making some changes while I was reviewing your codes

- [x] We have to make sure your page design isn't messed up. Reduce the font-size of the footer to match the original 
footer

- [x] Fix the missing image on the first article of *More in Space and Astronomy* section. 
Screen Shot 2020-09-27 at 6 32 00 PM


### Add these changes and submit another code review request.

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning:

### Hi team :wave:

**A lot of efforts have been put into this project. You've really done well to implement the changes required by previous reviewers. However, we still have to put one-to-two things in place before this project can be approved.**

- Let us improve on the `Editor's Pick` section. Looking at our clone (observe from the image below), the layout looks a bit different from the source.
Source (original):


Clone (ours):


**[ OPTIONAL ]**
- Still talking about looks, let's reduce the font size of the footer so it can be close to the source size.

Source (original):
 

Clone (ours):


**We're almost there: do not panic. Kindly make the changes above and request for another review when you are done.**

**You can reach me @taofeekolalere on slack.
Happy Coding :man_technologist:**

**Review by arslanbisharat — APPROVED** [review]
### Congratulations APPROVED :clap:

Hello Developers,

I can see you worked really hard on this project. and I'm very happy to approve this project with the latest changes you made in this project. :satisfied:
_The clone shows a good understanding and usage of HTML CSS💯_

- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
Now you can merge the branches and move forward with your curriculum. 🎆 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 

I am very excited to see your next project.

**Happy Coding**:smiley:
------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) slack: @arslanbisharat

**Pull request merged**